### DISCUSSION TITLE
Hi i have a questions to get right way immediately please help me

### DISCUSSION THREAD

**Discussion by Batgerel53** [saleor/saleor#8690] [Q&A]
**Question 1.** 

When will release full version of 3 saleor ? and we are using saleor 3.0.0 alpha. How can i upgrade new version with automatic?

 **Question 2.**
 Now im facing product api in graphql problem i cant send query of products api on graphql playground. Is it 3.0.0 problem?

 **Question 3.**
 Which one is best version of 3.0.0 using for any production now?

 **Question 4.**

n the dashboard i had to face problem with price on product section. It doesnt show any price on section i already put currency how can i fix this?

**Question 5. **
And can i create api to charge numbers units?

**Comment by karolkielecki** [comment]
@Batgerel53 For new projects, we recommend using the latest 3.0 beta version, There is no release date atm. 
Re graphql playground -- have you assigned any products to your channels? 
Re dashboard, please provide screenshots - I'm not sure if I understood correctly.
Re question 5 -- Could you please provide more info? Not sure what you meant.
